Formula/libvirt.rb

```diff
@@ -1,8 +1,9 @@
 class Libvirt < Formula
   desc "C virtualization API"
   homepage "https://www.libvirt.org"
-  url "https://libvirt.org/sources/libvirt-1.3.4.tar.gz"
-  sha256 "e2396ebebb3f3fdb50429ce8faa99559f6e8e3cc0493d5fa0c1999db189c25bd"
+  url "https://libvirt.org/sources/libvirt-1.3.5.tar.gz"
+  sha256 "93a23c44eb431da46c9458f95a66e29c9b98e37515d44b6be09e75b35ec94ac8"
+  head "https://github.com/libvirt/libvirt.git"
```

Comment:
Upstream doesn't seem to keep a pre-generated `configure` script around so this won't actually work as-is.